rules/defaults.yaml

```diff
@@ -468,22 +478,46 @@
 		MaxHeightDelta: 3
 	ThrowsShrapnel:
 		Weapons: LargeDebris
-		Pieces: 2, 20
-		Range: 2c848, 7c73
+		Pieces: 4, 15
+		Range: 3c512, 5c73
+	ThrowsShrapnel@SmallDebris:
+		Weapons: SmallDebris
+		Pieces: 10, 30
```

Comment:
The amount worries me. CD is already a performance hog due to the sheer amount of debris spawned.